concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -236,6 +244,38 @@ def main():
     )
 
     # Test CASE 4B + 5. Requestor sends ForceSubtaskResultsResponse with SubtaskResultsAccepted
+    #  Step 1. Send ForceSubtaskResults
+    api_request(
+        cluster_url,
+        'send',
+        PROVIDER_PRIVATE_KEY,
+        CONCENT_PUBLIC_KEY,
+        force_subtask_results(
+            timestamp = timestamp_to_isoformat(current_time),
+            ack_report_computed_task = ack_report_computed_task(
+                timestamp = timestamp_to_isoformat(current_time - (cluster_consts.subtask_verification_time * 1.4)),
+                subtask_id = task_id + '2C',
```

Comment:
Same here, please generate `task_id` once, at the beginning of test scenario.

Reply:
Done.